Microservices/Profile/pom.xml

```diff
@@ -36,6 +41,8 @@
 			<artifactId>spring-boot-starter-data-rest</artifactId>
 			<version>${spring.data.rest.version}</version>
 		</dependency>
+
```

Comment:
Just be careful of unintended whitespace (not really an issue in the grand run of things)